src/_TextInputWrapper.tsx

```diff
@@ -27,16 +27,22 @@ const sizeVariants = variant({
   prop: 'size',
   variants: {
     small: {
+      '--inner-action-size': '20px',
```

Comment:
Non-blocking suggestion for hard-coded size values: it would be nice if there were comments that identified the design token that the value comes from.